sycl/test-e2e/SubGroup/generic_reduce.cpp

```diff
@@ -1,6 +1,12 @@
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
 
+// SYCL 2020: Only the binary operators defined in Section 4.17.2 are supported
+// by the reduce functions in SYCL 2020, but the standard C++ syntax is used for
+// forward compatibility with future SYCL versions.
+//
+// REQUIRES: TEMPORARY_DISABLED
+
```

Comment:
@Pennycook , are you ok with this?

Reply:
I'm not sure I understand why it's being temporarily disabled.

Does `reduce_over_group` (currently) have different type constraints than the old extension, or something like that?

Reply:
From what I understand - yes.

Reply:
Then I'm okay with it.